groups/middleware/authorization.go

```diff
@@ -602,3 +702,22 @@ func (am *authorizationMiddleware) extAuthorize(ctx context.Context, extOp svcut
 
 	return nil
 }
+
+func (am *authorizationMiddleware) Callout(ctx context.Context, session authn.Session, op string, params map[string]interface{}) error {
+	pl := map[string]interface{}{
+		"entity_type": policies.GroupType,
+		"sender":      session.UserID,
+		"domain":      session.DomainID,
+		"time":        time.Now().String(),
+	}
+
+	for k, v := range params {
```

Comment:
Use `maps.Copy`. Be careful: we **must ensure params don't contain any of the pl values** so we don't overwrite them, so we should probably just do something like `maps.Copy(params, pl)` and call it with `params` instead of `pl`.